example/build.gradle

```diff
@@ -2,7 +2,12 @@ description = 'Hibernate Reactive Example'
 
 dependencies {
     implementation project(':hibernate-reactive-core')
+
+    // database drivers for PostgreSQL and MySQL
     runtimeOnly 'io.vertx:vertx-pg-client:3.9.1'
     runtimeOnly 'io.vertx:vertx-mysql-client:3.9.1'
+
+    // logging
+    runtimeOnly 'org.slf4j:slf4j-log4j12:1.7.30'
```

Comment:
just curious about the choice of slf4j + log4j ... why slf4j? Is vertx using it?

Reply:
Um ... dunno, it's what reactive core was set up to use. Wasn't my choice...

Reply:
ok, np I was just curious.